### ISSUE TITLE
Issue with Top and Bottom Toolbars Displayed Simultaneously in Default Browser Mode with Bottom Toolbar Enabled

### ISSUE THREAD

**Issue by Sylyia** [kiwibrowser/src.next#1056]
Title:
Issue with Top and Bottom Toolbars Displayed Simultaneously in Default Browser Mode with Bottom Toolbar Enabled

Bug Description:
When I set Kiwi Browser as the default browser and enable the bottom toolbar in the settings, third-party applications calling the browser result in both the top and bottom toolbars being displayed simultaneously. This issue occurs consistently.

Steps to Reproduce:

Set Kiwi Browser as the default browser.
Enable the bottom toolbar in the browser settings.
Have a third-party application initiate a call to open a URL in the default browser.
Expected Behavior:
The browser should only display the bottom toolbar when it is enabled in the settings. The top toolbar should not appear simultaneously.

Actual Behavior:
Both the top and bottom toolbars are displayed at the same time when a third-party application calls the browser.

Environment:

Device: [thyme]
Browser Version: 120.0.6099.116
Android Version: Android 13

![photo_2023-12-24_21-04-32](https://github.com/kiwibrowser/src.next/assets/54952917/49648ad2-99e3-4dba-ba3b-eb6e407c6351)

**Comment by Xisrr1** [comment]
Looks like this app is pretty much dead. This is a major bug and was not fixed in the past months.

**Comment by duyraven** [comment]
> Looks like this app is pretty much dead. This is a major bug and was not fixed in the past months.

Yes, this is disappointing. Another one of the few open source browsers for Android has died. This obvious problem lasted for at least a year before I reported it, and I waited for a long time before it was fixed before reporting it.

**Comment by obscuregit** [comment]
Kiwi browser 111.0.5563.104 (4474244826). I think this is the last version that works properly. It definitely needs to be fixed.

**Comment by Xisrr1** [comment]
> Kiwi browser 111.0.5563.104 (4474244826). I think this is the last version that works properly. It definitely needs to be fixed.

I'll just switch at this point. Cromite is a very good browser but its adblocker gets detected on many sites.
